backend/src/components/facilities.js

```diff
@@ -1,14 +1,56 @@
 'use strict'
 const { getOperation, patchOperationWithObjectId } = require('./utils')
 const { MappableObjectForFront, MappableObjectForBack } = require('../util/mapping/MappableObject')
-const { FacilityMappings, RoleMappings, UserMappings, UsersPermissionsFacilityMappings, LicenceMappings } = require('../util/mapping/Mappings')
+const { FacilityMappings, RoleMappings, UserMappings, UsersPermissionsFacilityMappings, LicenceMappings, FacilityAdditionalAddressMappings } = require('../util/mapping/Mappings')
 const HttpStatus = require('http-status-codes')
 
+/**
+ * Dynamics can take a selector with a list of fields
+ * Use this function to generate a list of fields based on mappings
+ */
+function getMappingString(mappings) {
+  let retVal = mappings
+    .map((item) => {
+      return item.back
+    })
+    .join(',')
+  return retVal
+}
+
+function makeThePayloadPretty(response) {
```

Comment:
How about just formatPayload or mapPayload/convertPayload?

Reply:
whoops I meant to change this before I pushed the code !! done